spec/support/models/stringified_symbol.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+# encoding: utf-8
+
+class StringifiedSymbol
+  include Mongoid::Document
+  store_in collection: "stringified_symbols", client: :other
+  field :stringified_symbol, type: StringifiedSymbol
```

Comment:
I suggest using a field name that isn't the same as the field type also, for example something like `:status` might be a good choice.